content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -160,15 +160,30 @@ Name, Organization, SAN, etc.) instead of the `kubeadm-config` ConfigMap. It is
 to keep them both in sync.
 {{< /note >}}
 
-`kubeadm certs renew` provides the following options:
+`kubeadm certs renew` can renew all certificates, using `kubeadm certs renew all` or can renew only specific certificates, as shown in the excerpt below of its command line help:
 
-- The Kubernetes certificates normally reach their expiration date after one year.
+```console
```

Comment:
```suggestion
```shell
```

Reply:
I'd like to push back on this suggestion - the style guide demonstrates in [Separate commands from output](https://github.com/kubernetes/website/edit/main/content/en/docs/contribute/style/style-guide.md) that commands should use \`\`\`shell, while their output should use \`\`\`console.

Excerpt from the style guide follows:

```
### Separate commands from output

Verify that the pod is running on your chosen node:

```shell
kubectl get pods --output=wide
```

The output is similar to this:

```console
NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
nginx    1/1       Running   0          13s    10.200.0.4   worker0
```

Reply:
Thank you, @neolit123!